flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliStrings.java

```diff
@@ -353,13 +354,19 @@ public static AttributedString messageWarning(String message) {
                 .toAttributedString();
     }
 
-    public static AttributedString messageError(String message, Throwable t) {
-        while (t.getCause() != null
-                && t.getCause().getMessage() != null
-                && !t.getCause().getMessage().isEmpty()) {
-            t = t.getCause();
+    public static AttributedString messageError(String message, Throwable t, boolean isVerbose) {
+        String expMessage;
+        if (isVerbose) {
+            expMessage = ExceptionUtils.getStackTrace(t);
```

Comment:
We should also get rid of empty cause to have better exception stack trace. The top cause may be `SqlExecutionException` which is unnecessary to print.